packages/perseus/src/__tests__/extract-perseus-data.test.ts

```diff
@@ -1,10 +1,158 @@
 import {describe, it, expect} from "@jest/globals";
 
+import {InputNumber, Radio, generateTestPerseusItem} from "..";
 import {
     getAnswersFromWidgets,
+    getCorrectAnswerForWidgetKey,
+    getValidWidgetKeys,
+    getWidgetTypeFromWidgetKey,
     injectWidgets,
+    isWidgetKeyInContent,
+    isWrongAnswerSupported,
+    shouldHaveIndividualAnswer,
 } from "../util/extract-perseus-data";
 
+const PerseusItemWithRadioWidget = generateTestPerseusItem({
```

Comment:
I wasn't sure if it was worth making an extract-perseus-data.data.ts file for just these two PerseusItems, but happy to do so if desired.